readthedocs/builds/forms.py

```diff
@@ -110,6 +110,7 @@ class Meta:
         model = RegexAutomationRule
         fields = [
             'description',
+            'on_match',
```

Comment:
I'd move this field (in the form) below to "Custom match:". I feel it's more natural to firstly write the regex and then make it negative/invert it.

Reply:
I don't agree, I think moving it down will make the user think the action or the tab/branch are inverted instead of th match.